cpp/src/plasma/lib/java/org_apache_arrow_plasma_PlasmaClientJNI.cc

```diff
@@ -0,0 +1,311 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Let's add this file to the C++ linting/clang-tidy

Reply:
I think this file has already in the C++ linting process now, because i just fix many linting errors which is feedback by the ci.